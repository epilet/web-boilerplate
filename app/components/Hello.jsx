import React from 'react';
import { inject, observer } from 'mobx-react';

@inject('greeting') @observer
export default class Hello extends React.Component {
  render() {
    const res = this.props.greeting.toName('Grace');
    return <h1>{res}</h1>;
  }
}

Hello.propTypes = {
  greeting: React.PropTypes.object,
};
