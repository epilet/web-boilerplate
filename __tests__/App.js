// Link.react-test.js
import React from 'react';
import Hello from '../app/components/Hello';
import renderer from 'react-test-renderer';

test('Testing app', () => {
  const component = renderer.create(
    <Hello />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
