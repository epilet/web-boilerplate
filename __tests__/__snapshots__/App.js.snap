exports[`test Testing app 1`] = `
<h1>
  Hello world
</h1>
`;
